From a72e226e20ab6e58290323439ffeca1ed733e801 Mon Sep 17 00:00:00 2001
From: Pawel Kurdybacha <pawel.kurdybacha@gmail.com>
Date: Sun, 22 Jan 2017 18:17:55 +0000
Subject: [PATCH 8/8] CMake static library output does not match ZeroMQ Config.

Configuration used in ZeroMQConfig.cmake.in does not match to
explicic "libzmq" output of static build.
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e351002..ddef056 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -908,10 +908,10 @@ if (MSVC)
   get_property(libzmq_pf TARGET libzmq PROPERTY ${U_CMAKE_BUILD_TYPE}_POSTFIX)
   set(libzmq_file libzmq${libzmq_pf}${CMAKE_LINK_LIBRARY_SUFFIX})
   get_property(libzmq_static_pf TARGET libzmq-static PROPERTY ${U_CMAKE_BUILD_TYPE}_POSTFIX)
-  set(libzmq_static_file libzmq-static${libzmq_static_pf}${CMAKE_LINK_LIBRARY_SUFFIX})
+  set(libzmq_static_file libzmq${libzmq_static_pf}${CMAKE_LINK_LIBRARY_SUFFIX})
 else()
   set(libzmq_file libzmq${CMAKE_SHARED_LIBRARY_SUFFIX})
-  set(libzmq_static_file libzmq-static${CMAKE_STATIC_LIBRARY_SUFFIX})
+  set(libzmq_static_file libzmq${CMAKE_STATIC_LIBRARY_SUFFIX})
 endif()
 
 # GNUInstallDirs "DATADIR" wrong here; CMake search path wants "share".
-- 
2.7.4

